processing/src/main/java/org/apache/druid/query/Query.java

```diff
@@ -124,22 +124,130 @@ default QueryContext getQueryContext()
     return null;
   }
 
-  <ContextType> ContextType getContextValue(String key);
```

Comment:
hmm, actually I guess this is an `@ExtensionPoint`, we should probably leave the old methods here to not be backwards incompatible

Reply:
Let me revert this.